src/components/Export.tsx

```diff
@@ -5,6 +5,7 @@ import SpinBox from "./SpinBox";
 import { RecordingOptions } from "../colorizer/RecordingControls";
 import { AppThemeContext } from "./AppStyle";
 import { CheckCircleOutlined } from "@ant-design/icons";
+import { clamp } from "three/src/math/MathUtils";
```

Comment:
Nit: Split imports into local dependencies vs node_modules dependencies -- helps readability.  I suspect there might be a Prettier setting for this but not sure?

Reply:
I couldn't find a prettier rule for this without installing a plugin, but there's an eslint rule for this, it looks like! If I enable it, it will, unfortunately, make me fix every import that's out of order across the whole project.

I'm going to add this as a TODO for now so I don't clutter up the PR, and I'll fix this instance manually 😅

Reply:
Thanks! Low priority but nice to have... 👍   (I feel like we have this in effect in at least one codebase but I might be mistaken)